update node version in webapp build env. The old version has problems
updating.

--- a/webapp/channels/build/Dockerfile
+++ b/webapp/channels/build/Dockerfile
@@ -1,3 +1,3 @@
-FROM node:14.16
+FROM node:18
 RUN apt-get update && apt-get install -y make gcc nasm libpng-dev
 CMD [ "node" ]
